src/EditorFeatures/CSharpTest/Diagnostics/NamingStyles/EditorConfigNamingStyleParserTests.cs

```diff
@@ -79,6 +79,31 @@ public static void TestPascalCaseRule()
             Assert.Equal(Capitalization.PascalCase, namingStyle.CapitalizationScheme);
         }
 
+        [Fact]
+        [WorkItem(40705, "https://github.com/dotnet/roslyn/issues/40705")]
+        public static void TestPascalCaseRuleWithKeyCapitalization()
+        {
+            var dictionary = new Dictionary<string, string>()
+            {
+                ["dotnet_naming_rule.methods_and_properties_must_be_pascal_case.severity"] = "warning",
+                ["dotnet_naming_rule.methods_and_properties_must_be_pascal_case.symbols"] = "Method_and_Property_symbols",
+                ["dotnet_naming_rule.methods_and_properties_must_be_pascal_case.style"] = "Pascal_Case_style",
+                ["dotnet_naming_symbols.method_and_property_symbols.applicable_kinds"] = "method,property",
+                ["dotnet_naming_symbols.method_and_property_symbols.applicable_accessibilities"] = "*",
+                ["dotnet_naming_style.pascal_case_style.capitalization"] = "pascal_case"
+            };
+            var result = ParseDictionary(dictionary);
+            Assert.Single(result.NamingRules);
```

Comment:
Assert.Single() returns the item, so you can combine this and the next line, ditto for everything else.